server/tests/integration/automation-scheduler.test.js

```diff
@@ -84,7 +84,8 @@ const getTestPlanReport = async (id, { transaction }) =>
                                 id
                             }
                             output
-                            assertionResults {
+                         assertionResults {
```

Comment:
Super tiny nit for formatting but the linter should be capturing. No need to update but something to look out for when we eventually update the linting rules